lib/watir-webdriver/locators/element/locator.rb

```diff
@@ -0,0 +1,249 @@
+module Watir
+  module Locators
+    class Element
+      class Locator
+        attr_reader :selector_builder
+        attr_reader :element_validator
+
+        WD_FINDERS = [
```

Comment:
Out of curiosity, is there a reason it is private? I guess `SearchContext` is a weird module to reference things from.